packages/oauth/oauth-provider-api/src/types.ts

```diff
@@ -0,0 +1,34 @@
+export type Account = {
+  sub: string
+  aud: string | [string, ...string[]]
+
+  email?: string
+  email_verified?: boolean
+  name?: string
+  preferred_username?: string
+  picture?: string
+}
+
+export type Session = {
+  account: Account
+  info?: never // Prevent relying on this in the frontend
```

Comment:
Why is that?

Reply:
Ah ok, so this `info` isn't intended to be even _sent_ to the frontend, got it. Maybe we eventually handle this differently so that `info` doesn't even exist on the type used by frontend clients.